Ev/Ev.php

```diff
@@ -845,7 +844,7 @@ public function at() {}
      *      numerical stability).
      * @param float $interval The current interval value. Can be modified any time, but changes only take effect when
      *      the periodic timer fires or EvPeriodic::again() is being called.
-     * @param callable $reschedule_cb If set, tt must return the next time to trigger, based on the passed time value
+     * @param null|callable $reschedule_cb If set, tt must return the next time to trigger, based on the passed time value
```

Comment:
The same as above

Reply:
`callable $reschedule_cb = null` still isn't nullable in `createStopped` function